.husky/pre-commit

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+. "$(dirname "$0")/_/husky.sh"
+
+if ! command -v vale &> /dev/null; then
```

Comment:
tell me more about this. Does it not come installed with NPM?

Reply:
Vale is not an npm package.